transactor/src/main/scala/io/mediachain/copycat/Client.scala

```diff
@@ -121,21 +122,37 @@ class Client(client: CopycatClient) extends JournalClient {
     stateListeners.foreach(_.onStateChange(Disconnected))
   }
   
+  private def recover() {
+    val cf = client.recover()
+    recoveryExecutor.submit(new Runnable {
+      def run {
+        try {
+          cf.join()
+          logger.info("Copycat session recovered")
+        } catch {
+          case e: InterruptedException => ()
```

Comment:
ok, I will skip the InterruptedException catch - I am fine with logging it if the client happens gets closed while recovering.